tools/clang/lib/CodeGen/CGExprConstant.cpp

```diff
@@ -840,10 +840,6 @@ class ConstExprEmitter :
           return SubExprResult;
         }
 
-        if (llvm::ConstantAggregateZero *CAZ = dyn_cast<llvm::ConstantAggregateZero>(SubExprResult)) {
```

Comment:
I have already moved this logic under else block below. So this is no longer needed.